cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst

```diff
@@ -0,0 +1,53 @@
+How to Add a Default Thumbnail For Unknown File Types
+=====================================================
+
+Quick Overview
+--------------
+
+**This cookbook is about a feature only provided in the Enterprise Edition.**
+
+In the asset management each file type has a default image but it's possible you want to add a filetype which has not.
```

Comment:
[...] image. Thus, if you add a new filetype, you might want to add a default one as well.